### DISCUSSION TITLE
workflow designer not displaying a CLI made workflow.

### DISCUSSION THREAD

**Discussion by guid-1178** [StackStorm/st2#5565] [UI] [answered]
Hi
I've created a orquesta workflow manually by editing the .yaml file and updating the action. 
i've found that when i want to use the visual editor to view the workflow. its just blank.

I've tried adding the positional comments that the visual workflow editor uses `# [39, 67]` but this doesn't seem to work.

I've been able to manually copy the yaml contents into the visual editor and it displays the workflow, it just seems to not be able to sync up with a cli made file.

Is there a step i need to take to make the visual editor show the workflow?

Thanks :-)

**Comment by amanda11** [comment]
This usually happens if the workflow composer thinks the yaml is incorrect in some way.  It doesn't need the positional comments, so that won't be the problem.

If you are using Chrome, then if you go to Developer Tools and take a look at the console you might see what it is complaining about in the workflow. (Or there should be an equivalent in other browsers).

When you say it works manually if you cut and paste the yaml into the visual editor, in that case did you create the workflow in the UI as a new workflow with the +, and then cut/pasted the yaml in? 

If you can do it that way, then I'd compare the difference between the actions/<workflowname>.meta.yaml and actions/workflows/<workflow>.yaml when you created it this way, with when you generated it by hand. That might make you see the difference. It may be a problem either with the meta file or the workflow file itself.

**Reply by guid-1178** [reply]
Hi @amanda11 

Thank you for your help with this.

Its taken me a while to get back to this issue.

I think I've figured it out now. when i was updating the workflow i was doing `st2 action update <action.name> <meta file>`
what i didn't realize is that it looks like i need to restart the web-frontend in order to "re-register" the workflow with the editor.

i did a `sudo st2ctl reload` and after the workflow re-appeared in the editor. 

hopefully this helps others in the future!

**Comment by guid-1178** [comment, marked as answer]
My issue is resolved. 
Please see my reply to Amanda11 for more information.
https://github.com/StackStorm/st2/discussions/5565#discussioncomment-2590797
Thanks
